src/Bundle.js

```diff
@@ -453,4 +464,55 @@ export default class Bundle {
 
 		return ordered;
 	}
+
+	trace ( module, localName, es6 ) {
+		const importDeclaration = module.imports[ localName ];
```

Comment:
This has bothered me a bit. `module.imports` is used for both imports and re-exports. The linking (dare I say, binding?) I'm proposing in #71, avoids this issue. Consider:

```
export { foo as bar } from './foo1';
import foo from './foo2';

export function baz () {
  return foo(4, []);
}; 
```
Two identifiers `foo` are imported. Which one is in `module.imports`? Can't this cause a lot of issues down the road?